object CallError {
  1 match { case 1 => 1 }()
}
